### ISSUE TITLE
Duplicated C:\Program Files\Git\cmd in the PATH

### ISSUE THREAD

**Issue by frank-e** [git-for-windows/git#1457]
- [x] I didn't see an an [open](https://github.com/git-for-windows/git/issues?q=is%3Aopen) issue for my observation after installing v2.16.1, and this nit is unrelated to your "issue template" for bugs:

For my purposes the installer adds _C:\Program Files\Git\cmd;_ at the end of the PATH. However, the PATH (in the environment of the installer) already contains this directory.

Presumably you check the PATH as noted in the registry, and on my Windows 7 SP1 x64 box that is:

`%SystemRoot%\system32;%SystemRoot%;%SystemRoot%\System32\Wbem;%SystemRoot%\System32\WindowsPowerShell\v1.0;%CRUFT.REXX%;%CRUFT.CODE%;%CRUFT.VAIO%;%CRUFT.LIVE%;%CRUFT.PATH%;%CRUFT.MSQL%`

.The CRUFT.CODE in this gibberish is:

`C:\Program Files\Perforce;C:\Program Files\SlikSvn\bin;C:\Program Files\Git\cmd;C:\Program Files\Python35\Scripts;C:\Program Files\Python35;C:\ProgramData\Oracle\Java\javapath`

In other words, maybe you should try to expand all %ENVVAR% when you check the PATH in the registry. Or maybe not, if it's only me.

**Comment by dscho** [comment]
> this nit is unrelated to your "issue template" for bugs:

I don't think that assessment is correct.

> However, the PATH (in the environment of the installer) already contains this directory.

That is a bug in your system. Before Git for Windows is installed, it makes *no sense* for `PATH` to contain `C:\Program Files\Git\cmd`. It is the installer's job to add that path (or not, depending on the choices made in the installer wizard) to the `PATH` variable.

And just for future record: when contributing to Open Source, at least to Git for Windows, it is a wise idea to refrain from phrases like "you should". Telling others what they have to do often does have an adverse effect (and in this case, it did).

**Comment by frank-e** [comment]
No problem, of course I installed the new version over an older version, and of course this is my last comment or contribution here.
